src/components/navbar.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Please kindly do not disable linters, it's better to fix linters it helps to write a clean and good codebase 😉
- [ ] Also I noticed you disabled linters in your `children.js` file, kindly fix that as well 🙃 🙃